Python_Code_Tiktok_Chat_To_OSC

```diff
@@ -3,73 +3,66 @@ from pythonosc import udp_client
 from TikTokLive import TikTokLiveClient
 from TikTokLive.events import GiftEvent, CommentEvent
 
-# Variables to track the state of roses, galaxies, and crowns
-rose = 0
-galaxy = 0
-boop_toggle = 0
+class TikTokEventHandler:
+    def __init__(self, ip: str, port: int, username: str):
+        self.rose = 0
+        self.galaxy = 0
+        self.boop_toggle = 0
```

Comment:
I know this comes from the original code, but I do not see the use in it. It is only ever assigned and never read. Is there a reason for this?